web/server.js

```diff
@@ -32,7 +32,41 @@ module.exports = function(options) {
     port: options.port
   });
 
-  server.register(require('hapi-auth-cookie'), function(err) {
+  server.register([
+    require('hapi-auth-cookie'),
+    require('scooter'),
+    {
+      register: require('blankie'),
+      options: {
+        defaultSrc: [
+          '\'self\''
+        ],
+        styleSrc: [
+          '\'self\'',
+          'https://fonts.googleapis.com'
+        ],
+        imgSrc: [
+          '\'self\'',
+          'data:',
+          'https://www.google-analytics.com',
+          'http://www.google-analytics.com'
+        ],
+        childSrc: [
```

Comment:
well, according to MDN it'll fall back on default-src, which would be 'self'.

will we ever even want iframes?